code/datums/jobs/job/shipside.dm

```diff
@@ -86,8 +86,9 @@ Make the TGMC proud!"})
 	shoes = /obj/item/clothing/shoes/marine
 	gloves = /obj/item/clothing/gloves/marine/officer
 	head = /obj/item/clothing/head/tgmcberet/fc
+	glasses = /obj/item/clothing/glasses/night/m56_goggles
```

Comment:
strongly disagree, dont just ignore #balance lol

Reply:
No one but you said FCs with smartgoggles was bad. Not sure what you are referring to.